match/match.go

```diff
@@ -106,6 +100,16 @@ func main() {
 		log.Fatal(err)
 	}
 
+	// Prometheus metrics
+	promPort, ok := config.Ports["prometheus"]
+	if !ok {
+		log.Fatal(err)
```

Comment:
Do we want to give a more specific error, or is this message understandable enough?

Reply:
Ha! The err here isn't even valid - good spot